base/database/batch.go

```diff
@@ -1,8 +1,8 @@
 package database
 // This file was adapted from https://github.com/bombsimon/gorm-bulk
 import (
-	"errors"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
This errors package contains methods for attaching messages, call stacks and wrapping errors. I want to unify error handling by using this package, and wrapping errors received from foreign code (attaching call stacks), instead of logging every error manually at the place where it occurred.

I'll make a longer write-up with PR for this later.